src/Resources/js/modules/participation-update-handler.ts

```diff
@@ -160,4 +164,68 @@ export class ParticipationUpdateHandler {
         this.updateCheckBoxWrapper($checkbox);
         this.toggleTooltip($checkbox, TooltipLabel.OFFERED_MEAL);
     }
-}
\ No newline at end of file
+
+    private static getCombinedMealDishes($meal: JQuery): Dish[] {
```

Comment:
This is a duplicate of ParticipationPreToggleHandler. After merge, we should move all the get.. and updates.. related to combined dishes to their on (static) service.